diff --git a/node_modules/@lexical/react/LexicalAutoFormatterPlugin.d.ts b/node_modules/@lexical/react/LexicalAutoFormatterPlugin.d.ts
index 5cd0793..a1ca233 100644
--- a/node_modules/@lexical/react/LexicalAutoFormatterPlugin.d.ts
+++ b/node_modules/@lexical/react/LexicalAutoFormatterPlugin.d.ts
@@ -6,4 +6,4 @@
  *
  */
 
-export default function LexicalAutoFormatterPlugin(): React.ReactNode;
+export default function LexicalAutoFormatterPlugin(): JSX.Element;
diff --git a/node_modules/@lexical/react/LexicalAutoLinkPlugin.d.ts b/node_modules/@lexical/react/LexicalAutoLinkPlugin.d.ts
index 20bee3e..908c088 100644
--- a/node_modules/@lexical/react/LexicalAutoLinkPlugin.d.ts
+++ b/node_modules/@lexical/react/LexicalAutoLinkPlugin.d.ts
@@ -17,4 +17,4 @@ export type LinkMatcher = (text: string) => LinkMatcherResult | null;
 export default function LexicalAutoLinkPlugin(props: {
   matchers: Array<LinkMatcher>;
   onChange?: ChangeHandler;
-}): React.ReactNode;
+}): JSX.Element;
