diff --git a/node_modules/@lexical/react/LexicalComposer.d.ts b/node_modules/@lexical/react/LexicalComposer.d.ts
index 123777e..0b2747c 100644
--- a/node_modules/@lexical/react/LexicalComposer.d.ts
+++ b/node_modules/@lexical/react/LexicalComposer.d.ts
@@ -19,4 +19,4 @@ type Props = {
   };
   children: React.ReactNode;
 };
-export default function LexicalComposer(arg0: Props): React.ReactNode;
+export default function LexicalComposer(arg0: Props): JSX.Element
